mapflow/mapflow.py

```diff
@@ -1792,6 +1800,23 @@ def submit_processing_rating_callback(self, response: QNetworkReply) -> None:
         )
         self.update_processing_current_rating()
 
+    def enable_rating_submit_button(self, radio_checked: Optional[bool]) -> None:
+        status_ok = False
+        radio_button_checked = False
+        row = self.dlg.processingsTable.currentRow()
+        if not row >= 0:
+            return
+        if self.dlg.processingsTable.item(row, self.config.PROCESSING_TABLE_STATUS_COLUMN_INDEX).text() == 'OK':
+            status_ok = True
+        for radio_button in self.radio_buttons:
+            if radio_button.isChecked():
+                radio_button_checked = True
+        self.dlg.ratingSubmitButton.setEnabled(status_ok and radio_button_checked)
+        if status_ok and radio_button_checked:
+            self.dlg.ratingSubmitButton.setToolTip("")
+        else:
+            self.dlg.ratingSubmitButton.setToolTip(self.tr("Please select processing and rating to submit"))
```

Comment:
Add please another tooltip for processing that is not in OK status
"Only correctly finished processings (status OK) can be rated" or something like this.

Reply:
Added